node/type.go

```diff
@@ -10,6 +10,9 @@ const (
 	// Light is a stripped-down Celestia Node which aims to be lightweight while preserving highest possible
 	// security guarantees.
 	Light
+	// Dev is a full-featured Celestia Node running alongside a mock embedded Core node process,
+	// designed to simulate block production (for testing/dev purposes only).
```

Comment:
This only runs in-memory, with nothing written to disk, right? Can you document a bit more the nuances here of running in `Dev` mode?

Reply:
Here and on the cli.